aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Assets.Scripts.Tiles;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Assets.Scripts.WFC
+{
+    /// <summary>
+    /// Represents the grid placer.
+    /// </summary>
+    public class GridPlacer : MonoBehaviour
+    {
+        /// <summary>
+        /// Represents the grid.
+        /// </summary>
+        private Grid _grid;
+
+        /// <summary>
+        /// Represents the room objects.
+        /// </summary>
+        public RoomObjects RoomObjects;
+
+        /// <summary>
+        /// Awake is called when the script instance is being loaded.
+        /// </summary>
+        public void Awake()
+        {
+            _grid = new Grid(5, 5);
```

Comment:
Should this be hardcoded?

Reply:
For now yes, because there are other stories that need the grid ASAP. This will be removed when WFC is implemented.